src/app/credential-issuer/middleware.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     try {
       logger.info("calling cri return lambda", { req, res });
       const apiResponse = await axios.post(
-        `${API_BASE_URL}${API_CRI_RETURN_PATH}`,
+        `${API_BASE_URL}${API_CRI_ACCESS_TOKEN_PATH}`,
```

Comment:
Might be worth updating the wording of that logging just above?

Reply:
Yep good shout